### PR TITLE
I added color customization

### PR DESCRIPTION
**Pull request by Sellsword9**
Honestly I just saw this repository somewhere, took a look at it, and decided to give it a try. Then I ended up doing one small nitpick (cargo clean instead of rm -rf target, i don't like having a file containing any kind of rm -rf anywhere on my pc/the user's ones). Then I went deeper on the code and run it. I addded an arg option for the color, which displays correctly. 

### **Disclaimer**: 
Sometimes the buffer overflows: Tried --mode snow --color black and it didn't work, something tried accessing a non-existant position in the buffer. If this is fine to you or you can&want to fix it, accept the PR. If you want all commits to be "100% release-ready", then be aware of this possible bug.
I have not been able to reproduce it since that, so I do not know how or why that happened. Maybe it had nothing to do with colors, in which case this should be an issue.

### CODE REVIEW

**Comment by ckaznable** [comment]
Thanks for PR, can you add a description of the options in the Usage section of the README?

There's also the overflow problem, which shouldn't be happening, and I may try to fix that in the future.

File: src/widget/timer.rs [hunk 1]
```diff
@@ -87,21 +87,21 @@ static ASCII_9: [u8; 25] = [
 pub struct Timer(pub crate::app::Timer);
 
 impl Timer {
-    fn render_colon(area: Rect, buf: &mut Buffer) {
+    fn render_colon(area: Rect, buf: &mut Buffer, color: Color) {
```
Comment (ckaznable) [inline src/widget/timer.rs]: Please place the color after the area

File: src/widget/timer.rs [hunk 2]
```diff
@@ -87,21 +87,21 @@ static ASCII_9: [u8; 25] = [
...
         let top = area.top();
 
-        buf.get_mut(left + 1, top + 1).set_char(COLON_CHAR).set_fg(Color::Reset);
-        buf.get_mut(left + 1, top + 3).set_char(COLON_CHAR).set_fg(Color::Reset);
+        buf.get_mut(left + 1, top + 1).set_char(COLON_CHAR).set_fg(color);
+        buf.get_mut(left + 1, top + 3).set_char(COLON_CHAR).set_fg(color);
     }
 
-    fn render_decimal(d: u8, area: Rect, buf: &mut Buffer) {
+    fn render_decimal(d: u8, area: Rect, buf: &mut Buffer, color: Color) {
```
Comment (ckaznable) [inline src/widget/timer.rs]: Please place the color after the area

File: src/cli.rs [hunk 3]
```diff
@@ -15,6 +17,9 @@ pub struct Args {
     /// effect level, The lower, the stronger [4-1000]
     #[arg(short, long, value_parser = level_range, default_value_t = 50)]
     pub level: u16,
+
+    #[arg(long, value_parser = Color::from_str, default_value = "white")]
+    pub color: Color,
```
Comment (ckaznable) [inline src/cli.rs]: Add a note describing the accepted values

**Review by ckaznable — CHANGES_REQUESTED** [review]

File: src/widget/timer.rs [hunk 4]
```diff
@@ -87,21 +87,21 @@ static ASCII_9: [u8; 25] = [
...
         let layout = Layout::new(Direction::Horizontal, Constraint::from_lengths([5, 1, 5])).split(area);
-        Self::render_number(d / 10, layout[0], buf);
-        Self::render_number(d % 10, layout[2], buf);
+        Self::render_number(d / 10, layout[0], buf, color);
+        Self::render_number(d % 10, layout[2], buf, color);
     }
 
-    fn render_number(number: u8, area: Rect, buf: &mut Buffer) {
+    fn render_number(number: u8, area: Rect, buf: &mut Buffer, color: Color) {
```
Comment (ckaznable) [inline src/widget/timer.rs:104]: Please place the color after the area

File: src/cli.rs (hunk 3 above)
Comment (ckaznable) [reply src/cli.rs]: Maybe rename it to timer_color? I may want to customize the color for weather in the future as well.

**Comment by Sellsword9** [comment]
That should be it

**Comment by ckaznable** [comment]
LGTM, Thanks.

**Pull request merged**